docs/tutorials/substreams/getting-started.experimental.md

```diff
@@ -0,0 +1,374 @@
+---
+title: 'Getting Started'
+
+order: -9999
+outline: [0,4]
+---
+
+# Substreams Quickstart
+
+The goal of this document is to get a development environment set up in as little time as possible.
+
+Here is a quick introduction to [Substreams from StreamingFast](https://substreams.streamingfast.io/).
+
+## Prerequisites
+
+Before deploying, please make sure you have all these tools installed.
+
+- `helm` is the tool for downloading a chart and installing it to your target cluster.
+- `minikube` is the tool for setting up a cluster in your local machine. This cluster will have all the basic functions of any other cloud solution.
+- `kubectl` is the tool for managing all the deployments/pods/services from your cluster which includes some useful functions like monitoring, deploying, debugging, ...
```

Comment:
I haven't seen the descrptions `monitoring, deploying, debugging, ...` , usuallly `monitoring, deploying, debugging, etc.`